packages/domain/src/entities/PublicationReport.ts

```diff
@@ -1,6 +1,4 @@
-// No entity was required so far for this use case.
-
-export enum ReportReason {
+export enum PublicationReportReason {
```

Comment:
We need to be careful with this change and provide an alias for backwards compatibility. At least in the immediate.

Reply:
it is here https://github.com/lens-protocol/lens-sdk/pull/816/files#diff-0ce407683ae3d6eabed3d60f01c2638a14a06f03e80931119ee70845bcd8def1R9-R16